packages/forma-36-react-components/src/components/IconButton/IconButton.test.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, configure } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
+configure({ testIdAttribute: 'data-test-id' });
```

Comment:
We should probably start using this everywhere then.

Reply:
yeah, let's clean this up as a follow up